config/example-extras.cfg

```diff
@@ -1965,7 +1967,7 @@
 #analog_range_kill_pin:
 #   The resistance range for a 'kill' button. Range minimum and maximum
 #   comma-separated values must be provided when using analog button.
-
+#
```

Comment:
Thanks.  This one formatting change does not look correct.  Otherwise, the change looks good to me.

-Kevin